tensorboard/plugins/graph/README.md

```diff
@@ -438,7 +438,11 @@ Note: building a Hierarchy modifies the SlimGraph's OpNodes in place!
 
 ### Rendering the view
 
-TODO
```

Comment:
Since there's at least in theory a lot more to say about Rendering the view than just the XLA cluster part, I'd maybe keep this line, so like

```
### Rendering the view

TODO: Elaborate here.

#### XLA Cluster Visualization

....
```

Reply:
Added: `TODO: Elaborate here for all rendering options.`